### PR TITLE
Upgrading Jet templating engine to V6

### PR DESCRIPTION
**Pull request by malusev998**

### CODE REVIEW

**Comment by Fenny** [comment]
@malusev998 Hi, I was about to merge it. Any reason why you closed it :p?

**Comment by malusev998** [comment]
I made a mass, renamed the mod file, because i needed the changes quickly, but when i changed my master branch it updated the pull request

**Comment by malusev998** [comment]
Mod file reverted

**Comment by malusev998** [comment]
@Fenny Any response?

**Review by anton7r — APPROVED** [review]

**Comment by ReneWerner87** [comment]
I will test your adjustments in the next few days and give you feedback.

File: jet/jet.go [hunk 1]
```diff
@@ -150,41 +176,32 @@ func (e *Engine) Load() error {
...
 		}
+
+		l.Set(name, string(buf))
 		// Create new template associated with the current one
 		// This enable use to invoke other templates {{ template .. }}
-		if _, err = e.Templates.LoadTemplate(name, string(buf)); err != nil {
-			return err
-		}
-		// Debugging
-		if e.debug {
```
Comment (ReneWerner87) [inline jet/jet.go:177]: can you please leave that in

**Review by ReneWerner87 — CHANGES_REQUESTED** [review]
very good work i have tested everything, only the one debug exclamation should stay in, if you adjust that i will merge it directly

File: jet/jet.go (hunk 1 above)
Comment (malusev998) [reply jet/jet.go:177]: Leave the print statement in when debugging is enabled?

Comment (ReneWerner87) [reply jet/jet.go:177]: exactly, the debug currently only activates the dev-mode and there you have no information about the rendered templates(i tested it), think that is quite useful

Comment (malusev998) [reply jet/jet.go:177]: I'll fix it later today or tomorrow, np

Comment (ReneWerner87) [reply jet/jet.go:177]: fixed

**Review by ReneWerner87 — APPROVED** [review]

**Pull request merged**